packages/backend/src/telemetry/privacy.test.ts

```diff
@@ -0,0 +1,106 @@
+import { privacy } from './privacy'
+describe('Privacy functions', () => {
+  const clipParams = {
+    minBucketSize: 1000,
+    maxBucketSize: 10000
+  }
+
+  let originalModule: typeof privacy
+
+  beforeEach(() => {
+    originalModule = { ...privacy }
+
+    jest.mock('./privacy', () => ({
+      ...originalModule,
+      applyPrivacy: jest.fn()
+    }))
```

Comment:
why do we need this mock here?

Reply:
that is mocking the module, but keeping it's implementation for all other methods except for applyPrivacy.
That is so that i can properly mock return values.

Reply:
leftover. thx